--- a/src/device.c
+++ b/src/device.c
@@ -1641,11 +1641,11 @@
 
   g_debug ("verify_cb: result %s", name);
 
-  if (priv->login_claim && error && error->domain == FP_DEVICE_RETRY)
+  if (error && error->domain == FP_DEVICE_RETRY)
     {
       g_clear_error (&error);
       error = g_error_new_literal (FP_DEVICE_ERROR, FP_DEVICE_ERROR_GENERAL,
-                                   "Login attempt ended; use password");
+                                   "Fingerprint attempt ended; use password");
     }
 
   if (priv->login_claim)
@@ -1707,11 +1707,11 @@
 
   g_debug ("identify_cb: result %s", name);
 
-  if (priv->login_claim && error && error->domain == FP_DEVICE_RETRY)
+  if (error && error->domain == FP_DEVICE_RETRY)
     {
       g_clear_error (&error);
       error = g_error_new_literal (FP_DEVICE_ERROR, FP_DEVICE_ERROR_GENERAL,
-                                   "Login attempt ended; use password");
+                                   "Fingerprint attempt ended; use password");
     }
 
   if (priv->login_claim)
